Source/ACE.Server/WorldObjects/Player_Database.cs

```diff
@@ -82,6 +102,8 @@ public void SavePlayerToDatabase()
 
         public void SaveCharacterToDatabase()
         {
+            if (DoNotSave) { return; }
```

Comment:
Character should still be saved right?

Doesn't God only affect the biota?

Reply:
Yes I believe only ignoring the biota could work. I thought it might be better to not save anything instead of saving some things and not others.

Reply:
What all is contained in character vs biota? I tried settings like /day and side by side vitals, even with this check in there the settings were retained - as a god.

Reply:
removed